types/underscore/underscore-tests.ts

```diff
@@ -2978,6 +2978,17 @@ _.defer(() => alert("deferred")); // $ExpectType void
     welcome("moe"); // $ExpectType any
 }
 
+// restArguments
+
+// Returns a version of the function that, when called, receives all arguments from and beyond startIndex collected into a single array.
+{
+    const raceResults = _.restArguments((words: string[]) => words.join(''));
+    raceResults("Dopey", "Grumpy", "Happy", "Sneezy", "Bashful", "Sleepy", "Doc");
+    const startIndex = 2;
+    const raceHasStartIndexResults = _.restArguments((words: string[]) => words.join(''), startIndex);
+    raceHasStartIndexResults("Dopey", "Grumpy", "Happy", "Sneezy", "Bashful", "Sleepy", "Doc");
```

Comment:
This still isn't how `startIndex` works. Please have another good look at the example code in the documentation. How many arguments does the function that is passed to `restArguments` have? How are those arguments sourced and how are they used?

Reply:
> This still isn't how works. Please have another good look at the example code in the documentation. How many arguments does the function that is passed to have? How are those arguments sourced and how are they used?`startIndex``restArguments`

I refer to this document: https://github.com/jashkenas/underscore/blob/master/test/functions.js, the startIndex parameter seems to be used this way.

Reply:
"Used this way" as in, passed as the last argument to `restArguments`. However, note that in that test, `startIndex` is *less* than the number of parameters of the function in the first argument. In your test, `startIndex` is *greater* than the number of parameters, which makes no sense. Please study https://underscorejs.org/#restArguments to understand why this matters.